### DISCUSSION TITLE
Welcome to the Terraform 30-Day Challenge!

### DISCUSSION THREAD

**Discussion by chiche-ds** [chiche-ds/30-Day-Terraform-challenge-#5] [General]
Hi everyone!

Welcome to the Terraform 30-Day Challenge! 🎉 I’m excited to see all of you here and can’t wait to start this journey together.

Feel free to use this space to:
- Introduce yourself
- Share your goals for the challenge
- Fun facts about you, your country, your neighbourhood
- Discuss anything related to Terraform that doesn’t fit into other categories

Let’s make this a great learning experience for everyone. Looking forward to connecting with all of you!

Best,
@chiche-ds

**Comment by kevshakes** [comment]
Happy to be here @chiche-ds 

My goal is to learn new skills and get certified in Terraform. 

I come from Kenya, the home of greatness in East Africa and one of the countries buzzing with tech in sub-saharan Africa and beyond. 

Let's connect on LinkedIn and Medium where I will be documenting my thoughts from this journey: 

L: https://linkedin.com/in/kevintuei

M: https://kevintuei.medium.com

**Reply by chiche-ds** [reply]
It’s always a pleasure working with you thanks for joining

**Comment by Walter-Obrien** [comment]
Hello, @chiche-ds, and thanks once again for this amazing opportunity.

I'm extremely happy to be part of this challenge and this growing community as well. I decided to hop into this because I knew it would be THE place for me to learn Terraform, master it, and get certified.

I'm a mechanical manufacturing teacher and AI Trainer from Cameroon who decided to transition completely into tech. And, as we're starting today, I'll be sharing my journey and documenting my work on the platforms below:

LinkedIn: https://www.linkedin.com/in/hermann-essoh-30800115a/
Medium: https://medium.com/@essohfranckhermann
Dev.to: https://dev.to/hermann_essoh

**Reply by chiche-ds** [reply]
Thanks for sharing can’t wait to check out your write up

**Comment by ITTARGET23** [comment]
Thank you, @chiche-ds, is a privilege to up skills
I am Ngayi Franklin

My ultimate goal is to gain the hands-on experience and master the core concepts in terraform. While preparing for the certification at the end of this challenge thank you. 

I am here because I didn’t know anything and ready to learn everything, and such opportunities is rare in my country and my neighborhood has very bad internet connection and bad road but that wouldn’t hinder my goal for this challenge.

**Reply by chiche-ds** [reply]
Our Goal here is to learn keep doing the baby steps by then end u will be a terraform guru

**Comment by sunilkumarald** [comment]
Hi @chiche-ds and everyone,

Thanks for the challenge.

I am from India and I am on a career transformation journey. Currently, I am in technical training and academics.
 
My goal is to get certified in Terraform after attending the challenge which I think will help me in my journey. 

Also, I am preparing for cloud certifications. 

I have some previous basic hands-on with Terraform.

I like to create technical content on YouTube, medium, and LinkedIn. Also, I have a WordPress website.

YouTube channel: https://www.youtube.com/@digibodh
LinkedIn: https://www.linkedin.com/in/sunilkumardigi/ (let's connect)
Medium: https://medium.com/@suniel.vns 

Thanks

**Reply by chiche-ds** [reply]
Glad to have you join us it’s always inspiring working with you

**Comment by Gaturu** [comment]
Hi @chiche-ds and everyone,

I'm really excited to be here and can't wait to dive into Terraform and pick up some solid, hands-on skills. I'm also looking forward to meeting and connecting with others who are just as passionate about this stuff as I am.

LinkedIn: https://www.linkedin.com/in/duncangaturu/
Medium: https://medium.com/@gaturugaturu

**Reply by chiche-ds** [reply]
Thanks for sharing checking your medium  right now

**Comment by Assan-zidan** [comment]
Hello everyone,
I am very hapy to be here.
I am so excited to start learning and dive into Terraform skills with Terraform 30-Day Challenge. Thanks 👍 

LinkedIn: https://www.linkedin.com/in/assan-zidan-nsangou/
X: https://x.com/assan_zidan
blog: https://assanzidan.dev/articles

@Assan-zidan

**Reply by chiche-ds** [reply]
Thanks for sharing your welcome to the challenge

**Comment by Adaeze-69** [comment]
Hi @chiche-ds ,

I was able to do my task for today and [here](https://www.linkedin.com/posts/adaeze-comfort-nnamdi_30daytfchallenge-hug-hashicorp-activity-7230694445904535552-No8L?utm_source=share&utm_medium=member_desktop) is the link to my LinkedIn post on it and to my [twitter](https://x.com/adaeze_nnamdi4/status/1824936580894621844) post too.

I was also able to purchase the hands-on labs and past questions on Udemy too. Went through the textbook on introduction to Terraform.

**Comment by nadeemtaj371** [comment]
Hi @chiche-ds and everyone,
My name is Nadeem Taj. I am passionate about Cloud Computing and DevOps tools. I am proficient in AWS, Linux, Docker, and git. The goal of joining this challenge is to Master in Terraform and get Certified. A fun fact about me is that I struggle with challenges. 

I love reading over watching tutorials.  I am interested in open-source but would love to contribute. Let's collab.  

Best regards 
Nadeem Taj

**Comment by sunilkumarald** [comment]
Hi @chiche-ds 
I purchased both the hands-on lab course and practice tests. I started with the book also. 
pl find my LinkedIn post for kick-starting the challenge https://www.linkedin.com/posts/activity-7231367305455149056-ViGZ?utm_source=share&utm_medium=member_desktop

Soon I will share my blog link
thanks
